src/rgw/driver/rados/rgw_sal_rados.cc

```diff
@@ -2283,7 +2283,14 @@ int RadosObject::read_attrs(const DoutPrefixProvider* dpp, RGWRados::Object::Rea
   read_op.params.obj_size = &state.size;
   read_op.params.lastmod = &state.mtime;
 
-  return read_op.prepare(y, dpp);
+  int ret = read_op.prepare(y, dpp);
+
+  RGWObjStateManifest *sm = rados_ctx->get_state(get_obj());
+  if (sm != nullptr) {
+    state.objv_tracker = sm->state.objv_tracker;
```

Comment:
`RGWObjectCtx::get_state()` has to acquire a mutex. can we add a objv_tracker pointer to `read_op.params` so that `prepare()` can handle this part?

Reply:
done